containers/Comments/CommentForm.styles.js

```diff
@@ -7,14 +7,20 @@ export const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     paddingHorizontal: SPACINGS.multiplyBy(1),
-    paddingVertical: SPACINGS.multiplyBy(0.75),
+    paddingVertical: SPACINGS.multiplyBy(1)
   },
   textInputContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
     flex: 2,
     backgroundColor: COLORS.paper,
     borderRadius: BORDERRADIUS.lg,
     padding: SPACINGS.multiplyBy(0.5),
   },
+  textInput: {
+    paddingTop: 0,
+    paddingBottom: 0
+  },
```

Comment:
this makes the clickable area of the text input very small clickable area should be at minimum 40x40

Reply:
for me it does not change the clicable area, if i do not set it to 0 it is not vertically centered.

Reply:
the padding doesn't do anything for me (looks the same with and without). however, the clickable area is only as wide as the placeholder text, clicking right of the placeholder text does nothing. Adding something like `width: '100%'` fixes it for me

Reply:
seems to be an iOS issue only, latest commit fixed it.